cms/search/tests/test_kafka_integration.py

```diff
@@ -8,9 +9,11 @@
 from cms.search.publishers import KafkaPublisher
 from cms.standard_pages.tests.factories import InformationPageFactory
 
+KAFKA_SERVER = os.environ.get("KAFKA_SERVER", "localhost:19092")
```

Comment:
question: why this change?

We have it `base.py` reading from the env. Let's leave it at that, and remove from `override_settings` below.

Reply:
Residue! Was testing something. Removed!